### DISCUSSION TITLE
UPDATE PLUGIN

### DISCUSSION THREAD

**Discussion by GaForgone** [GriefPrevention/GriefPrevention#1532] [Ideas]
Update the plugin to work with 1.17.1 servers

**Comment by Jikoo** [comment]
While I'm sure we'd all like to see an official release, if you look at the [commit log](https://github.com/TechFortress/GriefPrevention/commits/master) you can see multiple 1.17-specific changes being made. It's being worked on. If you're having problems and need a specific fix, dev builds are available [on the CI](https://ci.appveyor.com/project/RoboMWM39862/griefprevention/history).
